### DISCUSSION TITLE
Error in downloading bonsai vision design library

### DISCUSSION THREAD

**Discussion by heatherlien81** [bonsai-rx/bonsai#1636] [General]
Hi, I've just downloaded bonsai and tried to use it but the workflow is empty so I've diagnosed that the problem is that I haven't installed vision design library and dsp library. However, the package cannot be downloaded with the error as shown
<img width="249" alt="image" src="https://github.com/bonsai-rx/bonsai/assets/72308321/ceb050d5-6b8b-4b99-a57e-9d0a4c03b5ae">

**Comment by bruno-f-cruz** [comment]
uhm weird... I wonder if the MyGet Server was down. In any case, can you share your Bonsai.config file?
